tensorflow/core/kernels/mkl_requantize_per_channel_op.cc

```diff
@@ -120,16 +120,28 @@ class MklRequantizePerChannelOp : public OpKernel {
             const_cast<quint8*>(output->flat<quint8>().data()));
       }
 
-      std::unique_ptr<memory> input_mem_prim_(new memory(input_pd, input_buf));
-      std::unique_ptr<memory> output_mem_prim_(
-          new memory(output_pd, output_buf));
+      std::unique_ptr<memory> input_mem_prim(
+          new MEMORY_CONSTRUCTOR_USING_MD(input_md, cpu_engine_, input_buf));
+      std::unique_ptr<memory> output_mem_prim(
+          new MEMORY_CONSTRUCTOR_USING_MD(output_md, cpu_engine_, output_buf));
 
       mkldnn::reorder::primitive_desc reorder_pd =
-          mkldnn::reorder::primitive_desc(input_pd, output_pd, reorder_attr);
-      std::vector<mkldnn::primitive> net;
-      net.push_back(
-          mkldnn::reorder(reorder_pd, *input_mem_prim_, *output_mem_prim_));
-      stream(stream::kind::eager).submit(net).wait();
+          REORDER_PD_CONSTRUCTOR_WITH_ATTR(
+              GET_MEMORY_PRIMITIVE_DESC_FROM_MEM_PTR(input_mem_prim),
+              GET_MEMORY_PRIMITIVE_DESC_FROM_MEM_PTR(output_mem_prim),
+              cpu_engine_, reorder_attr);
+      mkldnn::stream reorder_stream = CPU_STREAM(cpu_engine_);
+#ifndef ENABLE_MKLDNN_V1
+      reorder_stream.submit(
+          {mkldnn::reorder(reorder_pd, *input_mem_prim, *output_mem_prim)});
+#else
+      std::unordered_map<int, mkldnn::memory> reorder_args = {
+          {MKLDNN_ARG_FROM, *input_mem_prim},
+          {MKLDNN_ARG_TO, *output_mem_prim}};
+      std::unique_ptr<mkldnn::primitive> reorder_prim(
+          new mkldnn::reorder(reorder_pd));
+      reorder_prim->execute(reorder_stream, reorder_args);
+#endif  // ENABLE_MKLDNN_V1
```

Comment:
Nit, since it's `#ifndef`.
```suggestion
#endif  // !ENABLE_MKLDNN_V1
```